### DISCUSSION TITLE
Dataset can not be downloaded.

### DISCUSSION THREAD

**Discussion by icekezuo** [logpai/loghub#45] [Q&A]
Hi fellows
     I am a big fan of this loghub. All the dataset are very important resources for system analysis. However all dataset are not available from the provided links. How to resolve it ?
     Thanks a lot.

**Comment by zhujiem** [comment]
Hi,

I tried the links just now. They still work fine. You may need VPN to
access zenodo?

On Wed, Mar 27, 2024 at 4:46 PM icekezuo ***@***.***> wrote:

> Hi fellows
> I am a big fan of this loghub. All the dataset are very important
> resources for system analysis. However all dataset are not available from
> the provided links. How to resolve it ?
> Thanks a lot.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/logpai/loghub/discussions/45>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABVU3I5CZXAMLLTSOLE6XHLY2KBN3AVCNFSM6AAAAABFKNTYECVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZWGQZDMNJYGQ>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
- ZJM

**Reply by icekezuo** [reply]
Thanks zhujiem,

It is finally done by edited the hosts file in my system !
